src/main/java/com/redhat/devtools/lsp4ij/LSPIJUtils.java

```diff
@@ -1221,4 +1221,39 @@ public static String getProjectUri(Project project) {
                 .map(l -> new Location(l.getTargetUri(), l.getTargetSelectionRange() != null ? l.getTargetSelectionRange() : l.getTargetRange()))
                 .toList();
     }
+
+    /**
+     * Finds the text range of the token at the specified offset in the provided file and editor.
+     *
+     * @param file   the file
+     * @param editor the editor
+     * @param offset the offset within the file
+     * @return the token at the specified offset in the file and editor, or null if no token is found
+     */
+    public static @Nullable TextRange getTokenTextRange(@NotNull PsiFile file, @NotNull Editor editor, int offset) {
```

Comment:
My first impression is that this method is similar to getWordRangeAt. I wonder if we could just using getWordRangeAt directly.